src/Particular.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -179,5 +179,13 @@ public static class DiagnosticDescriptors
             category: "Code",
             defaultSeverity: DiagnosticSeverity.Warning,
             isEnabledByDefault: true);
+
+        public static readonly DiagnosticDescriptor UseUtcNow = new DiagnosticDescriptor(
+            id: DiagnosticIds.UseUtcNow,
+            title: "Use DateTime.UtcNow or DateTimeOffset.UtcNow",
+            messageFormat: "Use {0}.UtcNow instead of {0}.Now.",
+            category: "Code",
+            defaultSeverity: DiagnosticSeverity.Error,
```

Comment:
Seems like a pretty heavy severity?

Reply:
I think it's a pretty heavy no-no. But anyway since we have TreatWarningsAsErrors the difference is really academic.

Reply:
Using DateTime.Now is perfectly valid for using in a UI like ServiceInsight.

Reply:
Then you suppress the error.